test/Microsoft.DotNet.Darc.Tests/DependencyRegistrationTests.cs

```diff
@@ -15,30 +14,31 @@ namespace Microsoft.DotNet.Darc.Tests;
 [TestFixture]
 public class DependencyRegistrationTests
 {
+    /// <summary>
+    /// Tests instantiating the operations
+    /// </summary>
     [Test]
-    public void AreDependenciesRegistered()
+    public void AreDarcOperationsRegistered()
     {
-        DependencyInjectionValidation.IsDependencyResolutionCoherent(services =>
+        foreach (Type operationType in Program.GetOptions().Concat(Program.GetVmrOptions()))
```

Comment:
shouldn't this be `optionType` or `options`, since we're getting it from the `GetOptions` method?